packager/media/formats/mp2t/adts_header.cc

```diff
@@ -63,7 +63,7 @@ bool AdtsHeader::Parse(const uint8_t* adts_frame, size_t adts_frame_size) {
   // Skip private stream bit.
   RCHECK(frame.SkipBits(1));
   RCHECK(frame.ReadBits(3, &channel_configuration_));
-  RCHECK((channel_configuration_ > 0) &&
+  RCHECK((channel_configuration_ >= 0) &&
```

Comment:
This condition can be removed as |channel_configuration_| is unsigned.

Reply:
Oh! So it is!